### PR TITLE
Update javafx-8-tutorial-zh-cn-part1.md

### PR DESCRIPTION
**Pull request by passacaglia**

### CODE REVIEW

File: collections/library/javafx-8-tutorial-zh-cn-part1.md [hunk 1]
```diff
@@ -188,7 +188,7 @@ languages:
 
 ## 创建主应用程序
 
-我们需要建立一个新的布局文件 `PersonOverview.fxml` 来做为主布局文件，它将包含一个菜单栏和你即将要显示的布局。
+我们需要建立一个新的布局文件 `RootLayout.fxml` 来做为主布局文件，它将包含一个菜单栏和你即将要显示的布局。
```
Comment (marcojakob) [inline collections/library/javafx-8-tutorial-zh-cn-part1.md]: I think this should stay PersonOverview.fxml

**Comment by marcojakob** [comment]
Thank you for the patch!

**Comment by passacaglia** [comment]
Yes, you're right. I should have seen the English first.

File: collections/library/javafx-8-tutorial-zh-cn-part1.md (hunk 1 above)
Comment (passacaglia) [inline collections/library/javafx-8-tutorial-zh-cn-part1.md]: Yes, you're right. I should have seen the English version first.Seems the translation is wrong. I'll send another pull request.

**Pull request merged**